Samples/HttpTrigger2/main.py

```diff
@@ -0,0 +1,5 @@
+import azurefuncs
```

Comment:
To align with all other Azure x Python packages, can we rename the module to `azure.functions` instead? 

Example - 
```python
import azure.functions as func
def main(req: func.HttpRequest):
        return 'Hello World!'
```

Reply:
Use functions instead for func